libs/localization/assets/locales/en/char_AratakiItto.json

```diff
@@ -1,7 +1,4 @@
 {
-  "a1": {
-    "name": "Consecutive Arataki Kesagiri uses"
-  },
```

Comment:
This is still in use 
https://github.com/frzyc/genshin-optimizer/blob/97b5208f8b36d5958e77da87631327965d60b4fb/apps/frontend/src/app/Data/Characters/AratakiItto/index.tsx#L274

Reply:
oh i reverted something i did but forgot to revert that ig oof